### PR TITLE
📚 Destinations V2: update typing & deduping doc link

### PR DESCRIPTION
**Pull request by cynthiaxyin**
Update `/understanding-airbyte/typing-deduping` with `/using-airbyte/core-concepts/typing-deduping`.

### CODE REVIEW

File: airbyte-integrations/connectors/destination-postgres/src/main/resources/spec.json [hunk 1]
```diff
@@ -218,7 +218,7 @@
       },
       "use_1s1t_format": {
         "type": "boolean",
-        "description": "(Early Access) Use <a href=\"https://docs.airbyte.com/understanding-airbyte/typing-deduping\" target=\"_blank\">Destinations V2</a>.",
+        "description": "(Early Access) Use <a href=\"https://docs.airbyte.com/using-airbyte/core-concepts/typing-deduping\" target=\"_blank\">Destinations V2</a>.",
```
Comment (cynthiaxyin) [inline airbyte-integrations/connectors/destination-postgres/src/main/resources/spec.json:221]: Not a strong opinion but probably not worth publishing new versions of Postgres + Postgres Strict Encrypt just for this; the next versions can just pick this up.

**Review by edgao — APPROVED** [review]
not publishing here seems fine. I have pending PRs targeting all three of these destinations so this will get published in the next day anyway

**Comment by octavia-approvington** [comment]
All in!! <br/> all in baby

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**